src/messaging/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from users.models import Profile
+# Create your models here.
+
+class Thread(models.Model):
+	profile_1 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_1')
+	profile_2 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_2')
```

Comment:
putting a nice static function that would be a queryset for a profile being in profile_1 or profile_2 here would be very good for code reusage in the future.